### PR TITLE
restore dev environment for rc2 development and deployment

### PR DESCRIPTION
**Pull request by PascalUlor**
#### What does this PR do?
Restore staging environment for continous deployment of RC2

#### Any background context you want to provide?

- [ ] I have performed a self-review of my own code
- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works .
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,2 @@
+REACT_APP_PRODUCTION_API=<Production Url>
```
Comment (attaradev) [inline .env.example]:
```suggestion
REACT_APP_API_URL=<API Url>
```

File: src/components/molecules/MediaIcons.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { Social } from "../atoms/SocialIcon";
 
-const baseUrl = process.env.REACT_APP_PRODUCTION_API || 'https://hackton-staging.herokuapp.com'
+const baseUrl = process.env.REACT_APP_STAGING_API || process.env.REACT_APP_PRODUCTION_API
```
Comment (attaradev) [inline src/components/molecules/MediaIcons.js]:
```suggestion
const baseUrl = process.env.REACT_APP_API_URL;
```

File: src/utils/api.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import Axios from "axios";
 
-const baseURL = process.env.REACT_APP_PRODUCTION_API;
+const baseURL = process.env.REACT_APP_STAGING_API || process.env.REACT_APP_PRODUCTION_API;
```
Comment (attaradev) [inline src/utils/api.js]:
```suggestion
const baseURL = process.env.REACT_APP_API_URL;
```

File: .env.example [hunk 4]
```diff
@@ -0,0 +1,2 @@
...
+REACT_APP_STAGING_API=<Staging Url>
```
Comment (attaradev) [inline .env.example]: We then delete the separate ENV variable for staging.

**Review by attaradev — CHANGES_REQUESTED** [review]
We can set the same ENV variable to different values based on the environment

File: .env.example (hunk 4 above)
Comment (PascalUlor) [reply .env.example]: The production and development urls are different and are both on netlify

**Review by attaradev — APPROVED** [review]

**Pull request merged**